fuse_constraints/include/fuse_constraints/relative_pose_2d_stamped_constraint.h

```diff
@@ -61,7 +61,7 @@ namespace fuse_constraints
 class RelativePose2DStampedConstraint : public fuse_core::Constraint
 {
 public:
-  SMART_PTR_DEFINITIONS(RelativePose2DStampedConstraint);
+  FUSE_CONSTRAINT_DEFINITIONS_WITH_EIGEN(RelativePose2DStampedConstraint);
```

Comment:
This could just be `FUSE_CONSTRAINT_DEFINITIONS(RelativePose2DStampedConstraint);`. Not alignment required for https://github.com/locusrobotics/fuse/pull/65/files#diff-f8a725e9aedd9304b2f3a77a8d873757R147